### PR TITLE
Re organise item drawer

### PR DESCRIPTION
**Pull request by shtukas**
This bring a new redesigned commissioning info for workflow items. 
Only UI work (HTML, CSS, and SVG changes).
Based on Akemi's design.

### CODE REVIEW

**Comment by shtukas** [comment]
Current (PROD) display:

current

**Comment by shtukas** [comment]
Akemi's design:

design

**Comment by shtukas** [comment]
New display:

new

File: public/components/content-list-drawer/_content-list-drawer.scss [hunk 1]
```diff
@@ -34,6 +34,109 @@ $drawer-breakpoint-small: 1400px;
...
+        }
+        &__element2 {
+          flex: 4 0 0;
+          height: 100%;
+          padding:12px;
+          border-right:2px solid $c-grey-300;
+        }
+        &__element3 {
+            flex: 1 0 0;
+            height: 100%;
```
Comment (philmcmahon) [inline public/components/content-list-drawer/_content-list-drawer.scss]: As this is the same for area2 you could avoid repeating it.

File: public/components/content-list-drawer/content-list-drawer.html [hunk 2]
```diff
@@ -2,259 +2,246 @@
...
-                            <img class="drawer__user-image" ng-if="assigneeImage.length > 1" ng-class="{ 'drawer__user-image--placeholder': !assigneeImage }" ng-src="{{ assigneeImage }}" height="38" width="38" alt="contentItem.item.assignee"/>
-                            <span class="drawer__section-data-row--editable drawer__user-name" ng-click="toggleAssigneeEditing();" onbeforesave="onBeforeSaveAssignee($data)">{{ contentItem.assignee || "None" }}</span>
-                        </div>
-                        <div ng-switch-when="true">
-                            <punters stub="contentItem" indrawer="true"></punters>
+            <div class="content-list-drawer__container-innerv2"><!-- div for padding/spacing -->
+                <div class="content-list-drawer__container-innerv2__area1">
+                    <div class="content-list-drawer__container-innerv2__area1__furniture">
+                        <div class="content-list-drawer__container-innerv2__area1__furniture__header">
+                            <i class="drawer__icon--content-type" wf-icon="furniture"/>
```
Comment (philmcmahon) [inline public/components/content-list-drawer/content-list-drawer.html]: as discussed- class should relate to icon - e.g. icon--furniture

File: public/components/content-list-drawer/_content-list-drawer.scss [hunk 3]
```diff
@@ -34,6 +34,109 @@ $drawer-breakpoint-small: 1400px;
     border: 3px solid $c-grey-200;
 }
 
+.content-list-drawer__container-innerv2 {
```
Comment (philmcmahon) [inline public/components/content-list-drawer/_content-list-drawer.scss]: v2 can go when the previous styles are removed

File: public/components/content-list-drawer/content-list-drawer.html [hunk 4]
```diff
@@ -2,259 +2,246 @@
...
-                    <li class="drawer__section">
-                        <span class="drawer__section-title">First published</span>
-                        <span class="drawer__section-data-row">{{ (contentItem.firstPublished | wfLocaliseDateTime:$root.globalSettings.location | wfFormatDateTime) || "-" }}</span>
-                    </li>
-                    <!--<li class="drawer__section">-->
-                        <!--<span class="drawer__section-title">Last published</span>-->
-                        <!--<span class="drawer__section-data-row">{{ contentItem.publishedTime || '-' }}</span>-->
-                    <!--</li>-->
-                </ul>
+            <!-- Pascal's Update -->
```
Comment (philmcmahon) [inline public/components/content-list-drawer/content-list-drawer.html]: 😄

Comment (shtukas) [inline public/components/content-list-drawer/content-list-drawer.html]: Corrected in 9bfabc4 :)

File: public/components/content-list-drawer/_content-list-drawer.scss [hunk 5]
```diff
@@ -34,6 +34,109 @@ $drawer-breakpoint-small: 1400px;
...
+                }
+            }
+        }
+    }
+    &__area2 {
+        display: flex;
+        flex-direction: row;
+        height: 100%;
+        background-color: $c-grey-200;
+        &__element1 {
```
Comment (shtukas) [inline public/components/content-list-drawer/_content-list-drawer.scss]: Done in 3eaa8fd ee8f3cf 1b2e401 and f8aef09

**Comment by shtukas** [comment]
Updated UI after corrections:

picture 1

**Comment by philmcmahon** [comment]
Looks great! 👍 👍 👍 

We should probably send an email round workflow users before deploying this to PROD, lets sort that today. Great work!

**Comment by shtukas** [comment]
Awesome ^_^

**Comment by shtukas** [comment]
Updated display after latest request. "Hover" behavior is now correct. 

picture 2

**Comment by shtukas** [comment]
Checked build #45 on CODE. All good.

**Comment by philmcmahon** [comment]
The view versions and resend to incopy text is still in blue - could you make them the same as close details? And I think the close details should always be in the bottom right. Otherwise 👍 !

**Comment by shtukas** [comment]
> incopy text is still in blue 

Colour correction has been applied. In both standard and hovering, so now only the background changes colour when being hovered. 

> And I think the close details should always be in the bottom right.

Totally agree with you ! I was following the original order, but the close button in last position feels much better from a user expectation point of view :)


picture 1

**Comment by shtukas** [comment]
Successfully tested on CODE.

**Pull request merged**